i3pystatus/cpu_freq.py

```diff
@@ -0,0 +1,54 @@
+"""
+The module gathers information by default from `/proc/cpuinfo` about the current cpu frequency
```

Comment:
This docstring should go into the class docstring below ; the docstring of Python modules is ignored.